disruption_py/core/utils/math.py

```diff
@@ -175,8 +187,9 @@ def gaussian_fit(*args):
     coeffs : array
         The coefficients of the fit.
     """
-
-    coeffs, *_ = curve_fit(gauss, *args)
+    with warnings.catch_warnings():
```

Comment:
I did not do this myself because I thought that it would have been safer to filter class (OptimizeWarning) but _also_ message.
at present scipy.curve_fit does raise only covariance-based OptimizeWarnings, but what if they add anything new in that class? we wouldn't ever see those warnings again.
hmm... is there a quick and easy way to filter message, too?